wagtail_localize/management/commands/export_po.py

```diff
@@ -0,0 +1,152 @@
+import pathlib
+
+import polib
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+from wagtail.core.models import Site
+
+from wagtail_localize.models import Locale, default_locale_id
+from wagtail_localize.workflow.views.translate import MessageExtractor
```

Comment:
Just to note that this code is just a stop-gap until I implement something proper at the model level. Happy for it to import these from views for now, but we will probably end up rewriting some of the logic here once the real API for extracting/ingesting strings has been developed

Reply:
Yes, I don't want to refactor all of that and I agree that importing stuff from the view is not a correct design.